### PR TITLE
Header Detailpage #64

### PR DESCRIPTION
**Pull request by sannevanseeventer**
# Header Detailpage #64 

## What does this change? 🚀 
Start for the component 'HeaderDetailPage'. This component will eventually be placed at the top of the Detailpage for WOGO. The code currently contains MockData. The component is therefore not yet complete, but I would appreciate receiving feedback on the HTML and CSS.

## How Has This Been Tested? ✅ 
- [x] User test
- [x] Accessibility test
- [x] Performance test
- [x] Responsive Design test
- [x] Device test
- [x] Browser test

### Small screens
Scherm­afbeelding 2024-05-17 om 11 25 30

### Large screens
Scherm­afbeelding 2024-05-17 om 11 25 03

## How to review 💡  
There are a few questions I would appreciate that could help me during a review:

1. Have the responsive images been applied correctly?
2. Due to the design of the text in the header, I'm unsure how to handle accessibility for screen readers. How can I ensure that a screen reader reads out the title & subtitle 'Regular Cocktail Walk' without mentioning 'route 1' in between? Or is there a more elegant way to do this?
3. General comments on how I can improve my code.

Thank you in advance!

### CODE REVIEW

File: src/lib/components/molecules/HeaderContent.svelte [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+<section class="content-container">
+  <!-- First content row -->
+  <div class="title-label">
+    <h1>{mockData[0].title}</h1>
+    <span class="label" aria-hidden="true">{mockData[0].label}</span>
+  </div>
+
+  <!-- Second content row -->
+  <h1 class="subtitle">{mockData[0].subtitle}</h1>
```
Comment (ju5tu5) [inline src/lib/components/molecules/HeaderContent.svelte:38]: Ony one ```<h1>``` element is allowed. I think you want a different heading level here..

File: src/lib/components/organisms/HeaderDetailPage.svelte [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<script>
+  import HeaderContent from '../molecules/HeaderContent.svelte'
+  import HeaderImage from '../atoms/HeaderImage.svelte'
+</script>
+
+<div class="header-container">
```
Comment (ju5tu5) [inline src/lib/components/organisms/HeaderDetailPage.svelte:6]: That's a lot of nested ```<div>```'s, is this really necessary or can you use a more semantic option instead..

File: src/routes/[slug]/+page.svelte [hunk 3]
```diff
@@ -9,12 +9,12 @@
   <kruimelpad parentSlug />
 } -->
 
-<div>
+<!-- <div>
```
Comment (ju5tu5) [inline src/routes/[slug]/+page.svelte:12]: Mag je gewoon verwijderen ;)

File: static/marker.svg
Comment (ju5tu5) [inline static/marker.svg:1]: Did you pull this through SVGOMG?

**Review by ju5tu5 — APPROVED** [review]
Nice and very readable code. I've made some remarks so if you can put them in issues then afterwards you can merge this branch.

**Comment by sannevanseeventer** [comment]
@ju5tu5 

Thanks for your feedback! Sorry, I don't understand what you mean by "afterwards you can merge this branch", because you already did or am I confused? And now, do I need to make a new branch to implement those remarks? 

Have a nice weekend!

**Pull request merged**